src/uwtools/config/formats/ini.py

```diff
@@ -1,10 +1,9 @@
+# pylint: disable=duplicate-code
 import configparser
 from io import StringIO
-from types import SimpleNamespace as ns
-from typing import Optional
 
 from uwtools.config.formats.base import Config
-from uwtools.config.support import depth
```

Comment:
It could be that `DEPTH` is still not _quite_ the right name -- maybe `REQUIRED_DEPTH` would be more explanatory. But, to answer your question: `DEPTH` is meant to define what the depth of each config format _must_ be, while `depth()` says what a given config's depth _is_. In fact, calling `depth()` probably really only makes sense for YAML configs at the moment, as that's the only format permitting variable depth. The rest are all fixed. But, if we ever needed to work with JSON, or with XML as an input instead of just as an output, `depth()` could be meaningful for them, too.